var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -102,6 +106,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
             "brig",
             "c",
             "c++",
+            "d",
```

Comment:
Does:
```suggestion
            conditional("d", when="@9.1:"),
```
work too? If it does (as it should) you can delete a bunch of conflicts.

Reply:
It looks like we could use `conditional` for several other `languages` too. Do you mind if we keep this part of the PR as is and I will come up with a follow-up PR that makes use of `conditional` whenever possible?

Reply:
No I don't.